packages/shared/src/graphql/posts.ts

```diff
@@ -28,6 +28,9 @@ export enum PostType {
   Welcome = 'welcome',
 }
 
+export type InternalReadType = PostType[];
```

Comment:
Export not needed here, and can actually also just type it directly as it's just the one type